migrations/0006_add_file_size.sql

```diff
@@ -1,4 +1,7 @@
 -- Add file_size column to pdf_files table
 -- This column stores the file size in bytes for statistics calculation
 
+-- SQLite doesn't have ALTER TABLE IF NOT EXISTS, but this will fail gracefully if column exists
+-- The migration system should handle the error and continue
+
 ALTER TABLE pdf_files ADD COLUMN file_size INTEGER DEFAULT 0; 
```

Comment:
i know i know something something cloudflare, but i think it's 100% fine for us still to blow up our tabes and start with new ones rather than migrating (fun fact: all of my stuff is broken all the time regardless, it's not like this is leading to clean organized dev)